app/controllers/admins_controller.rb

```diff
@@ -57,9 +58,9 @@ def permission_params
   end
 
   def user_params
-    { full_name: params[:full_name],
-      role: params[:role],
-      organization_id: params[:organization_id],
-      device_updated_at: Time.current }.compact
+    {full_name: params[:full_name],
+     role: params[:role],
+     organization_id: params[:organization_id],
+     device_updated_at: Time.current}.compact
```

Comment:
I'm not a fan of this indentation style yet. I much prefer:

```
{
  full_name: params[:full_name],
}
```

It adds lines, but I feel is so much more readable.